app/src/main/kotlin/com/wire/android/ui/authentication/login/sso/LoginSSO.kt

```diff
@@ -60,11 +63,17 @@ fun LoginSSOScreen(
     LoginSSOContent(
         scrollState = scrollState,
         loginSSOState = loginSSOState,
-        onCodeChange = { loginSSOViewModel.onSSOCodeChange(it) },
-        onDialogDismiss = { loginSSOViewModel.onDialogDismiss() },
-        onLoginButtonClick = suspend { loginSSOViewModel.login(serverConfig) },
-        scope = scope
+        onCodeChange = loginSSOViewModel::onSSOCodeChange,
+        onDialogDismiss = loginSSOViewModel::onDialogDismiss,
+        onRemoveDeviceOpen = loginSSOViewModel::onTooManyDevicesError,
+        //TODO: replace with retrieved ServerConfig from sso login
+        onLoginButtonClick = suspend { loginSSOViewModel.login(ServerConfig.STAGING) },
```

Comment:
Can we use `serverConfig` param from the `LoginSSOScreen` instead of hardcoding it here?